server/src/test/java/org/elasticsearch/search/geo/GeoBoundingBoxIT.java

```diff
@@ -225,7 +229,8 @@ public void testCompleteLonRange() throws Exception {
         assertThat(searchResponse.getHits().getTotalHits(), equalTo(1L));
         searchResponse = client().prepareSearch()
                 .setQuery(
-                        geoBoundingBoxQuery("location").setValidationMethod(GeoValidationMethod.COERCE).setCorners(50, 0, -50, 360).type("indexed")
+                        geoBoundingBoxQuery("location").setValidationMethod(GeoValidationMethod.COERCE).setCorners(50, 0, -50, 360)
+                        .type("indexed")
```

Comment:
Having this line on the same indention of as the one that builds the object that it is poking doesn't quite feel right to me.